docs/references/ayx-plugin-cli.md

```diff
@@ -0,0 +1,487 @@
+    (automatically installed with Python 3.8.5)
+
+Each SDK also has its own requirements, which are listed on each
+respective help page.
+
+We highly recommend that you develop your custom tools inside of a
+virtual environment (for example,
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+With Miniconda, you can create and activate a virtual environment with
+these commands:
+
+`conda create -n SDKEnv python=3.8.5`
+
+`conda activate SDKEnv`
+
+This creates an isolated development environment that minimizes the risk
+of creating a package that Alteryx Designer and other users won\'t be
+able to use. It does this by keeping the dependencies required by
+different projects separate.
+
+### Installation
+
+To install the AYX Plugin CLI, run `pip install ayx_plugin_cli`.
+
+#### Verify Installation
+
+To verify that you installed the CLI properly, run
+`ayx_plugin_cli version`.
+
+If the installation was successful this command returns the version
+number. If the CLI was not installed properly, the `ayx_plugin_cli`
+command will not be recognized in the terminal, and your terminal will
+reflect that.
+
+### CLI Versions
+
+To check the version of the CLI that is installed, run
+`ayx_plugin_cli version`.
+
+The CLI automatically checks for the newest version and alerts you if a
+new version is available.
+
+To upgrade the CLI version, run `pip install ayx_plugin_cli --upgrade`.
+
+#### CLI Version Support
+
+We will support the latest version of the CLI. Visit [Release
+Notes](https://help.alteryx.com/developer-help/ayx-python-sdk-release-notes) to review new features as well as fixed and known
+issues.
+
+### Uninstall
+
+To uninstall the CLI, run `pip uninstall ayx_plugin_cli`.
+
+Once you uninstall the AYX Plugin CLI, you will lose the ability to
+manage your custom tool projects. Uninstalling the CLI removes all of
+the features that help create, delete, and package custom tools into
+YXIs.
+
+
+
+# AYX Plugin CLI Commands
+
+Explore the [AYX Plugin
+CLI](https://help.alteryx.com/developer-help/ayx-plugin-cli) commands and parameters.
+
+## sdk-workspace-init
+
+This command initializes a workspace---a directory that contains all of
+the source code and configuration details for a set of Alteryx SDK
+plugins. Users create a workspace to house all of their plugin code, and
+once it\'s ready, generate a YXI for said code. The command creates an
+`ayx_workspace.json` file that defines package details and tools. It
+also generates a `backend/` directory for the plugin code, `config/`
+directory for plugin configurations, `ui/` directory for the plugin's UI
+components, a README, and a gitignore file.
+
+Note: Most ayx-plugin-cli commands expect an ayx_workspace.json file in
+the current directory, and throw an exception otherwise.
+
+
+### Parameters
+
+Note: If a parameter is not passed in as part of the CLI call, the CLI
+will wait for standard input for the ignored parameters, even if they're
+optional.
```

Comment:
TW review:

Note: If a parameter is not passed in as part of the CLI call, the CLI
waits for standard input for the ignored parameters, even if they're
optional.

Reply:
TODO: Next week in new MR @jcayx